
Title: My Second Blog
Date: 2019-09-12
Slug: blog2

This is my second blog post


```python
import pandas as pd
import chart
```


```python
df = pd.DataFrame([
    ['Ryan', 'https://duryan00.github.io/blog/'],
    ['Hock', 'https://teckhockng.github.io/blog/']
    ['Kavitha', 'https://kavitha-shashidharan.github.io/blog/']
    ['Anika', 'https://nawarani.github.io/blog/']
    ['Benoit', 'https://bolive84.github.io/Blog/']
])
```


    ---------------------------------------------------------------------------

    TypeError                                 Traceback (most recent call last)

    <ipython-input-14-1e0d49164eba> in <module>
          2     ['Ryan', 'https://duryan00.github.io/blog/'],
          3     ['Hock', 'https://teckhockng.github.io/blog/']
    ----> 4     ['Kavitha', 'https://kavitha-shashidharan.github.io/blog/']
          5     ['Anika', 'https://nawarani.github.io/blog/']
          6     ['Benoit', 'https://bolive84.github.io/Blog/']
    

    TypeError: list indices must be integers or slices, not tuple



```python

```


```python

```


```python

```
